plugins/woocommerce/client/legacy/css/twenty-nineteen.scss

```diff
@@ -506,7 +506,23 @@ table.variations {
 			}
 		}
 
+		.comment-form-rating {
+			@include reset-rating-fieldset();
+
+			fieldset {
```

Comment:
Once the `fieldset` HTML element was added, each Twenty* theme required adjustments to preserve the old visual appearance.